### ISSUE TITLE
ValueError: Unknown format code 'd' for object of type 'float'

### ISSUE THREAD

**Issue by 132nd-etcher** [132nd-vWing/ESST#106]
https://sentry.io/etcher/esst/issues/511219039/

```
ValueError: Unknown format code 'd' for object of type 'float'
(9 additional frame(s) were not displayed)
...
  File "asyncio\base_events.py", line 467, in run_until_complete
    return future.result()
  File "esst\dcs\dcs.py", line 316, in run
    await core.CTX.loop.run_in_executor(None, missions_manager.initial_setup)
  File "concurrent\futures\thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "esst\dcs\missions_manager.py", line 279, in initial_setup
    metar = emiz.weather.mizfile.get_metar_from_mission(str(mission.path))
  File "emiz\weather\mizfile\mizfile_get_metar.py", line 104, in get_metar_from_mission
    visibility = ('{:04d}M'.format(visibility))

ValueError: Unknown format code 'd' for object of type 'float'
```

**Comment by 132nd-etcher** [comment]
Fixed in https://github.com/132nd-etcher/emiz/commit/e3abee05849861fb35b2ce4b2b79ac036b21a242

**Comment by 132nd-etcher** [comment]
Fixed in #107
